### PR TITLE
more careful filtering

### PR DESCRIPTION
**Pull request by yingjieMiao**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/RelatedLibraryCommander.scala [hunk 1]
```diff
@@ -50,17 +50,18 @@ class RelatedLibraryCommanderImpl @Inject() (
...
-        case RelatedLibrary(lib, kind) =>
-          !fakeUsers.contains(lib.ownerId) &&
-            !userLibs.contains(lib.id.get) &&
-            !libQualityHelper.isBadLibraryName(lib.name)
-      }
+    def isUnwantedLibrary(relatedLib: RelatedLibrary): Boolean = {
+      val lib = relatedLib.library
+
+      lib.state == LibraryStates.INACTIVE ||
+        fakeUsers.contains(lib.ownerId) ||
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RelatedLibraryCommander.scala:57]: Ooo, this is quite expensive (1400 users).

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RelatedLibraryCommander.scala:57]: it's a Set, get from cache. should be ok.

**Comment by andrewconner** [comment]
Thanks. What's the number of keeps a library has to have to be recommended?

**Comment by yingjieMiao** [comment]
for related libraries, min keep size is 2. @eishay wanted that, so that user can explore more.
popular libraries or same owner library should have 5+ keeps.

**Comment by yingjieMiao** [comment]
Cortex side timely update library lda info. That library was turned to inactive almost immediately. However, the lib-lib related graph is NOT updated frequently, nor listening on any seqNum. It's a 12hour cron-job.

**Pull request merged**